.github/workflows/docs_build.yml

```diff
@@ -0,0 +1,26 @@
+name: Docs Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
This is not right. If we want to run this on all PRs we should remote lines 8 and 9.